src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import interfaces.AreaCalculator;
+import interfaces.Drawable;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure(String color) {
+        super();
```

Comment:
```suggestion
```
if you dont call super() or any other super(params), it will be called by default